### PR TITLE
SDK-1252. Management of "My Backups" folder

### PR DESCRIPTION
**Pull request by catalinr-m**

### CODE REVIEW

**Review by sergiohs84 — DISMISSED** [review]

**Comment by sergiohs84** [comment]
Let's wait for API to include this new attribute, @catalinr-m , so we can test that the returned JSON for `ug` command is parsed correctly. Afterwards, feel free to merge. Thanks

File: include/megaapi.h [hunk 1]
```diff
@@ -11779,6 +11782,43 @@ class MegaApi
...
+         *
+         * The associated request type with this request is MegaRequest::TYPE_SET_ATTR_USER
+         * Valid data in the MegaRequest object received on callbacks:
+         * - MegaRequest::getParamType - Returns the attribute type MegaApi::USER_ATTR_MY_BACKUPS_FOLDER
+         * - MegaRequest::getFlag - Returns false
+         * - MegaRequest::getNodehandle - Returns the provided node handle
+         * - MegaRequest::getMegaStringMap - Returns a MegaStringMap.
+         * The key "h" in the map contains the nodehandle specified as parameter encoded in B64
+         *
+         * If the folder is not private to the current ccount, or is in Rubbish, or is in a synced folder,
```
Comment (catalinr-m) [inline include/megaapi.h:11796]:
```suggestion
         * If the folder is not private to the current account, or is in Rubbish, or is in a synced folder,
```

**Comment by sergiohs84** [comment]
trigger compilation

**Review by sergiohs84 — APPROVED** [review]

**Comment by sergiohs84** [comment]
trigger compilation

**Comment by sergiohs84** [comment]
The failed Jenkins task is unrelated to this PR, but an issue in API for heartbeats (already solved in staging). We can proceed with merge

**Pull request merged**